.circleci/config.yml

```
@@ -0,0 +1,18 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/node:lts
+    working_directory: ~/repo
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - v1-dependencies-{{ checksum "package.json" }}
```

Just reading this https://circleci.com/docs/2.0/yarn which indicates we should change to this:

```suggestion
            - v1-dependencies-{{ checksum "yarn.lock" }}
```

I also don't think we need the next line?

---

You're right! I took most of this from their docs, sounds like they need to update them! The line afterwards is a fallback, I'll add a comment in to make it clear what its used for